helpers/calendar/ratios_and_utilization.py

```diff
@@ -167,23 +166,20 @@ def get_bookings_analytics_count(self, query, start, end):
         end_dt = dateutil.parser.parse(day_after_end_date)
         number_of_days = (end_dt - start_dt).days
 
-        if number_of_days <= 15:
+        if number_of_days <= 30:
             dates = CommonAnalytics.get_list_of_dates(start, number_of_days)  # noqa E501
             for date in dates:
                 bookings = CommonAnalytics.get_total_bookings(self, query, date[0], date[1])  # noqa E501
                 string_date = dateutil.parser.parse(date[0]).strftime("%b %d %Y")  # noqa E501
                 output = BookingsAnalyticsCount(period=string_date, bookings=bookings) # noqa E501
                 results.append(output)
 
-        elif number_of_days >= 90:
+        else:
             dates = CommonAnalytics.get_list_of_month_dates(start_date, start_dt, day_after_end_date, end_dt)  # noqa E501
             for date in dates:
                 bookings = CommonAnalytics.get_total_bookings(self, query, date[0], date[1])  # noqa E501
                 string_month = dateutil.parser.parse(date[0]).strftime("%B")
                 output = BookingsAnalyticsCount(period=string_month, bookings=bookings) # noqa E501
                 results.append(output)
```

Comment:
I feel like since we are not limiting a user to a specific date, we should just have one function i.e a function to show us the analytics for any given date. What I mean is we should maybe not care if the data we get is monthly or for 3 days so I think we should do away with the `if statement`. What do you think?

Reply:
This is because of the date range is greater than a month the data should not be returned as daily data but rather monthly data. This will ensure the fronted have data that they can easily display.

Reply:
> This is because of the date range is greater than a month the data should not be returned as daily data but rather monthly data. This will ensure the fronted have data that they can easily display.

Awesome clarification